.github/workflows/test-EMULATOR.yml

```diff
@@ -3,14 +3,27 @@ name: OSW-EMULATOR-test
 on:
   workflow_dispatch:
   push:
+    paths_ignore: '["**/*.md", "**/scripts/screen_capture/**"]'
   pull_request:
     branches: [ master, develop ]
 
 jobs:
+  check_skip:
+    runs-on: ubuntu-latest
+    outputs:
+      should_skip: ${{ steps.skip_check.outputs.should_skip }}
+    steps:
+      - id: skip_check
+        uses: fkirc/skip-duplicate-actions@master
```

Comment:
Do not use master here - please pin it to a specific version number or even commit hash instead!